h2/src/main/org/h2/expression/function/Function.java

```diff
@@ -2291,6 +2302,18 @@ private static Value oraHash(Value value, long bucket, long seed) {
         return ValueBigint.get((hc & Long.MAX_VALUE) % (bucket + 1));
     }
 
+    private static Value base64encode(byte[] data, String algorithm) {
+        final EncodingAlgorithm algorithmImpl = EncodingFactory.getEncodingAlgorithm(algorithm);
+        final String result = algorithmImpl.encode(data);
+        return ValueVarchar.get(result);
+    }
```

Comment:
Please, inline everything into this method, three additional classes for a such small function aren't needed.